test/seneca.test.js

```diff
@@ -1200,7 +1200,7 @@ describe('seneca', function () {
             // --seneca.log.all and count INs
             // ... | grep act | grep IN | wc -l
             // sensitive to changes in plugin init and internal action calls
-            assert.equal('{ calls: 15, done: 15, fails: 0, cache: 1 }',
+            assert.equal('{ calls: 14, done: 14, fails: 0, cache: 1 }',
```

Comment:
no actually it's a very good test and catches unseen breakage to the start up sequence